test_unstructured_ingest/expected-structured-output/confluence-diff/MFS/1540126.json

```diff
@@ -327,7 +327,7 @@
         "eng"
       ],
       "page_number": 1,
-      "text_as_html": "<table><br><tbody><br><tr><td>Milestone</td><td>Owner</td><td>Deadline</td><td>Status</td></tr><br><tr><td>         </td><td>     </td><td>        </td><td>      </td></tr><br><tr><td>         </td><td>     </td><td>        </td><td>      </td></tr><br><tr><td>         </td><td>     </td><td>        </td><td>      </td></tr><br></tbody><br></table>"
+      "text_as_html": "<table><tr><td>Milestone</td><td>Owner</td><td>Deadline</td><td>Status</td></tr><tr></tr><tr></tr><tr></tr></table>"
```

Comment:
This one is also interesting because it produces empty rows (rows with no cells). Those rows also appeared in the prior case but with empty cells.

We should probably suppress emission of rows with no cells. I'll add that in.